packages/frontend/Dockerfile

```
@@ -56,4 +56,5 @@ COPY --from=builder /home/rafiki/packages/frontend/build ./packages/frontend/bui
 COPY --from=builder /home/rafiki/packages/frontend/public ./packages/frontend/public
 
 WORKDIR /home/rafiki/packages/frontend
-CMD ["sh", "./node_modules/.bin/remix-serve", "build"]
\ No newline at end of file
+
+CMD ["sh", "-c", "NODE_ENV=production ENABLE_INSECURE_MESSAGE_COOKIE=true ./node_modules/.bin/remix-serve ./build/index.js"]
```

I think the `ENABLE_INSECURE_MESSAGE_COOKIE` should be passed as a build argument in the compose file. `ENABLE_INSECURE_MESSAGE_COOKIE` will be `true` when starting the frontend application every time (for everyone that is going to use the Rafiki's frontend image), since we use this specific Dockerfile to build the image.

https://docs.docker.com/compose/compose-file/compose-file-v3/#args

```suggestion
ARG ENABLE_INSECURE_MESSAGE_COOKIE=false # default value if the argument is not present at build time

ENV NODE_ENV=production
ENV ENABLE_INSECURE_MESSAGE_COOKIE=$ENABLE_INSECURE_MESSAGE_COOKIE

CMD ["sh", "-c", "./node_modules/.bin/remix-serve ./build/index.js"]
```

I experimented with this method and it appears to be effective for me. I would greatly appreciate any additional feedback on it to see if it works for everyone since I'm not entirely sure about this approach.

---

The above approach might not even be necessary, we can just get rid of the inline environment variables:

```suggestion
CMD ["sh", "-c", "./node_modules/.bin/remix-serve ./build/index.js"]
```